osquery/tables/system/darwin/filesystems.cpp

```diff
@@ -0,0 +1,48 @@
+#include <sys/param.h>
+#include <sys/ucred.h>
+#include <sys/mount.h>
+
+#include <glog/logging.h>
+
+#include <osquery/tables.h>
+
+namespace osquery {
+namespace tables {
+
+  QueryData genFilesystems(QueryContext& context) {
```

Comment:
We usually don't indent the namespace body, but clang-format will take care of things like that.